native_client/ctcdecode/scorer.cpp

```diff
@@ -123,18 +172,10 @@ void Scorer::reset_params(float alpha, float beta) {
   this->beta = beta;
 }
 
-std::string Scorer::vec2str(const std::vector<int>& input) {
```

Comment:
Same, maybe we should avoid removing dead code to make it easier to update in the future ?